fix(admin/AI): Small additions (#11907)
nextcloud/documentation: admin_manual/ai/app_stt_whisper2.rst

The *stt_whisper2* app is one of the apps that provide Speech-To-Text functionality in Nextcloud and act as a media transcription backend for the :ref:`Nextcloud Assistant app<ai-app-assistant>`, the *talk* app and :ref:`other apps making use of the core Translation API<stt-consumer-apps>`. The *stt_whisper2* app specifically runs only open source models and does so entirely on-premises. Nextcloud can provide customer support upon request, please talk to your account manager for the possibilities.

> This app supports input and output in other languages than English, if the underlying model supports the language.

Reviewer: ```suggestion
This app supports input and output in languages other than English if the underlying model supports the language.
```

This app uses `faster-whisper <https://github.com/SYSTRAN/faster-whisper>`_ under the hood. Output quality will differ depending on which model you use, we recommend the following models:

 * OpenAI Whisper large-v2 or v3 (multilingual)

>  * OpenAI Whisper medium.en (only supports English)

Reviewer: ```suggestion
 * OpenAI Whisper medium.en (English only)
```

Supplying alternate models
~~~~~~~~~~~~~~~~~~~~~~~~~~

> This app allows supplying alternate models in the ``/nc_app_llm2_data`` directory of the docker container. You can use any `*faster-whisper* model by Systran on hugging face <https://huggingface.co/Systran>`_ by simply

Reviewer: ```suggestion
This app allows supplying alternate models in the ``/nc_app_llm2_data`` directory of the docker container. You can use any `*faster-whisper* model by Systran on hugging face <https://huggingface.co/Systran>`_ in the following way:
```
